client/src/app/shared/shared-video-miniature/video-list-header.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Inject } from '@angular/core'
+
+export abstract class GenericHeaderComponent {
+  constructor (@Inject('data') public data: any) {}
+}
+
+@Component({
+  selector: 'h1',
```

Comment:
Not sure to understand this `h1`

Reply:
it is just the outer tag for the component

Reply:
`h1`? But it's already an existing html tag. We should create dedicated selector with `my-` prefix